packages/design-system/src/components/Status/variations/StatusWarning.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import i18n from 'i18next';
+import StatusPrimitive, { StatusProps } from '../Primitive/StatusPrimitive';
+
+export type StatusWarningProps = Omit<StatusProps, 'icon' | 'variant' | 'inProgress'>;
```

Comment:
variant seems always to have same props, isn't it?

Reply:
Yes, at this point it does. Not sure what the best practice would be.

Reply:
I like the explicitness of it. Each variant is a single file with types etc.. Yes those are the same types here in this context, but it may not be for other components. It feel better to enforce a single file pattern early on.